styles.css

```diff
@@ -378,7 +378,7 @@ li {
 }
 
 .project-x-title {
```

Comment:
[OPTIONAL] As mentioned in the requirements:

![req](https://user-images.githubusercontent.com/111214189/229315260-58440198-d447-442f-9a3c-7aec7497b4d2.png)

kindly make sure you implemented the corresponding button interactions 👍 . This was missed by the previous reviewer, but I highly recommend you implement this change. Your portfolio will look much better and it will give a better user experience 💯 .